src/acl_kernel.cpp

```diff
@@ -2785,12 +2785,21 @@ l_copy_and_adjust_arguments_for_device(cl_kernel kernel, cl_device_id device,
     // Exclude kernel argument value from device-side buffer by default.
     cl_uint buf_incr = 0;
 
-    if (arg_info->addr_space == ACL_ARG_ADDR_LOCAL) {
+    if (arg_info->streaming_info) {
+#ifdef MEM_DEBUG_MSG
+      printf("streaming");
+#endif
+      stream_info.args.emplace_back(aocl_mmd_streaming_kernel_arg_info_t{
+          .name = arg_info->streaming_info->interface_name.c_str(),
```

Comment:
https://github.com/intel/fpga-runtime-for-opencl/runs/6283308609?check_suite_focus=true
```
src\acl_kernel.cpp(2793): error C7555: use of designated initializers requires at least '/std:c++20'
```